lib/crawly/worker.ex

```diff
@@ -117,7 +117,25 @@ defmodule Crawly.Worker do
              result: {:ok, next} | {:error, term()}
   defp parse_item({response, spider_name}) do
     try do
-      parsed_item = spider_name.parse_item(response)
+      # get parsers
+      parsers = Crawly.Utils.get_settings(:parsers, spider_name, nil)
+
+      parsed_item =
+        if parsers != nil do
+          case Crawly.Utils.pipe(parsers, %{}, %{
+                 spider_name: spider_name,
+                 response: response
+               }) do
+            {false, _} ->
```

Comment:
Do we drop the entire item if one of the parsers returns false?

Reply:
Will add in logging for now. maybe we can add in an :on_parsed_item_drop_callback